grassroots-shared/src/dtos/PhoneCanvass/CallStatus.dto.ts

```diff
@@ -20,3 +25,20 @@ export function CallStatusDecorator(): PropertyDecorator {
     ApiProperty({ enum: CallStatusEnum }),
   );
 }
+
+enum CallResultEnum {
+  CANCELED = "CANCELED",
+  COMPLETED = "COMPLETED",
```

Comment:
I don't love "COMPLETED" being both a result and a status, but it aligns with what twilio does.

Reply:
what do COMPLETED and FAILED mean in this case? Maybe a link to the twilio docs would be helpful on this enum?

Reply:
Yeah, my original implementation actually did lump them together. 

I think they are actually conceptually pretty distinct though. The state machine doesn't care why a call ended, it just needs to know that it did. 

We only need to know the result for logging / statistics purposes. 

I kinda like splitting them out for that reason.

Also, with the way I've implemented the state transitions, it would mean either a bit more complexity (one class with multiple possible states) or a bunch more boilerplate. 

The conceptual distinction is a much stronger reason though. :P